package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "regon-prog",
+  "version": "1.0.0",
+  "description": "REGON generator",
+  "main": "regon.js",
+  "scripts": {
+    "start": "node regon.js",
+    "test": "node node_modules/jasmine/bin/jasmine.js spec/RegonSpecs.js"
```

Comment:
Only tests for regon will be executed. Please enable execution of the tests for nip generator as well